include/libtorrent/alert_types.hpp

```diff
@@ -3000,6 +3003,30 @@ TORRENT_VERSION_NAMESPACE_3_END
 		file_index_t reserved;
 	};
 
+	// this alert is posted when two separate torrents (magnet links) resolve to
```

Comment:
It may be helpful to specify that the "first" or "owner" `torrent_handle` is the one that received the metadata. This is "obvious" but may be helpful to have guaranteed in documentation, in case the app wants to use this information when resolving the conflict.

Reply:
Yeah, that's true. Since that torrent doesn't have the metadata assigned to it, but left in its original state, I figured that it won't matter. But I suppose it may convey information about which hash has a swarm for instance